documentation/SupportingDocuments/Population Density API - Algorithm Initial Proposal.md

```diff
@@ -0,0 +1,33 @@
+# Population Density Data API
+## Algorithm initial proposal
+### Inputs for the algorithm:
+* UEs connection records data is received pseudonymized or hashed.
+* Requested Space is received in the API customers’ request.
+### Process is as follows:
+<ol>
+<li><strong>Cleanup</strong>: Repeated records will be deleted and traffic events associated with M2M lines are filtered due to they do not contribute to the process of estimating future population density.<br>
```

Comment:
The important step is missing (it should be step 1 I think): which data is processed? The algorithm processes some records, but which records? The whole history? Two years? 1 Year? Last hour?
If I understood our discussion correctly the steps must be like.
1) identify day of the week and time frame for which the prediction is requested
2) read location information for the same weekday and time frame for the recent 4 weeks
3) clean up and other current steps

Reply:
In step one has been included the time-frame of the records: one year of historical information.

And we illustrate the prediction (step 6) with the step 1 and 2 you mentioned "The algorithm identifies a day of the week and time frame for which the prediction is requested. Then, it will read the records associated for the same weekday and time frame for the previous 4 weeks".